lesson16/index.js

```diff
@@ -0,0 +1,158 @@
+          element.classList.remove("is-active");
+        });
+      });
+    });
+
+    if (target.textContent === "news") {
+      console.log("news");
+
+      const imgWrapper = document.querySelector(".news_Img");
+      imgWrapper.innerHTML = "";
+      const img = document.createElement("img");
+      img.src = allData[0].image.img;
+      img.alt = allData[0].image.alt;
+      imgWrapper.appendChild(img);
+
+      renderNewsList(allData[0], lists);
+    } else if (event.target.textContent === "economy") {
+      console.log("economy");
+
+      const imgWrapper = document.querySelector(".news_Img");
+      imgWrapper.innerHTML = "";
+      const img = document.createElement("img");
+      img.src = allData[1].image.img;
+      img.alt = allData[1].image.alt;
+      imgWrapper.appendChild(img);
+
+      renderNewsList(allData[1], lists);
+    } else if (event.target.textContent === "sports") {
+      console.log("sports");
+
+      const imgWrapper = document.querySelector(".news_Img");
+      imgWrapper.innerHTML = "";
+      const img = document.createElement("img");
+      img.src = allData[2].image.img;
+      img.alt = allData[2].image.alt;
+      imgWrapper.appendChild(img);
+
+      renderNewsList(allData[2], lists);
+    }
+  });
+};
+
+const renderNewsList = (content, lists) => {
+  const newsList = document.querySelector(".news_Lists");
+  newsList.innerHTML = "";
+  Object.keys(content.article).forEach((key) => {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const comment = document.createElement("span");
+    a.href = content.article[key].url;
+    a.textContent = content.article[key].title;
+    const newMark = document.createElement("span");
+    newMark.classList.add("newMark");
+    newMark.innerText = "new!";
+    const dateDiff = getDateDiff(content.article[key].publishedDate);
+    if (dateDiff < 3) {
+      a.appendChild(newMark);
+    }
+    const commentCount = content.article[key].commentCount;
+    if (commentCount > 0) {
+      const commentIcon = document.createElement("img");
+      commentIcon.src = "./lesson16/img/icon_comment.png";
+      a.appendChild(commentIcon);
+      comment.textContent = commentCount;
+    }
+    li.appendChild(a);
+    li.appendChild(comment);
+    lists.appendChild(li);
+  });
+};
+
+const getDateDiff = (publishedDate) => {
+  // const publishedDateValue = new Date(publishedDate);
+  const today = new Date(Date.now());
+  const date1 = new Date(publishedDate);
+  const termDay = (today - date1) / 86400000;
+  return Math.floor(termDay);
+};
+
+const getData = () => {
+  return new Promise((resolve) => {
+    try {
+      resolve(fetch(url));
+    } catch (error) {
+      console.error(error);
+    } finally {
+      console.log("finally");
+    }
+  });
+};
+
+const init = async (inputNum, inputName) => {
+  // createLoadingImage();
+  const response = await getData();
+  allData = await response.json();
```

Comment:
I don't think it is necessary to assign to a variable that is declared at the beginning of a line.
```suggestion
  const allData = await response.json();
```